src/muxy.ts

```diff
@@ -197,6 +199,22 @@ export class Muxy implements MuxyInterface {
   public messenger: Messenger;
 
   /**
+   * Internal {@link purchaseClient}.
```

Comment:
This `@link` would link to itself. Notice how `messenger` links to the `Messenger` class.